ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-redesigned.component.html

```diff
@@ -0,0 +1,208 @@
+<main class="main main_activity"
```

Comment:
- Extracted the html template into a separate file.
- Refactored in order to have more readable template and decrease code lines.
- Used (hidden/non-displayed) templates as much as possible (^ refactoring) and placed them below the main template.